x-pack/plugins/ml/public/application/components/import_export_jobs/export_jobs_flyout/export_job_warning_callout.tsx

```diff
@@ -0,0 +1,187 @@
+            }
+          >
+            <FilterJobList jobs={jobsWithFilters} />
+          </EuiAccordion>
+        )}
+      </EuiCallOut>
+
+      <EuiSpacer size="m" />
+    </>
+  );
+};
+
+const CalendarJobList: FC<{ jobs: JobDependencies }> = ({ jobs }) => (
+  <>
+    {jobs.length > 0 && (
+      <>
+        {jobs.map(({ jobId, calendarIds }) => (
+          <>
+            <EuiText size="s">
+              <h5>{jobId}</h5>
+              {calendarIds.length > 0 && (
+                <FormattedMessage
+                  id="xpack.ml.importExport.exportFlyout.exportJobDependenciesWarningCallout.calendarList"
+                  defaultMessage="{num, plural, one {calendar} other {calendars}}: {calendars}"
+                  values={{ num: calendarIds.length, calendars: calendarIds.join(', ') }}
+                />
+              )}
+            </EuiText>
+            <EuiSpacer size="s" />
+          </>
+        ))}
+      </>
+    )}
+  </>
+);
+
+const FilterJobList: FC<{ jobs: JobDependencies }> = ({ jobs }) => (
+  <>
+    {jobs.length > 0 && (
+      <>
+        {jobs.map(({ jobId, filterIds }) => (
+          <>
+            <EuiText size="s">
+              <h5>{jobId}</h5>
+              {filterIds.length > 0 && (
+                <FormattedMessage
+                  id="xpack.ml.importExport.exportFlyout.exportJobDependenciesWarningCallout.filterList"
+                  defaultMessage="{num, plural, one {filter} other {filters}}: {filters}"
+                  values={{ num: filterIds.length, filters: filterIds.join(', ') }}
+                />
+              )}
+            </EuiText>
+            <EuiSpacer size="s" />
+          </>
+        ))}
+      </>
+    )}
+  </>
+);
+
+function getTitle(jobs: JobDependencies, calendarCount: number, filterCount: number) {
+  if (calendarCount > 0 && filterCount === 0) {
+    return i18n.translate(
+      'xpack.ml.importExport.exportFlyout.exportJobDependenciesWarningCallout.calendarOnlyTitle',
+      {
+        defaultMessage:
+          '{jobCount, plural, one {# selected job has} other {# selected jobs have}} dependencies on {calendarCount, plural, one {a calendar} other {calendars}}',
+        values: { jobCount: jobs.length, calendarCount },
+      }
+    );
+  }
+
+  if (calendarCount === 0 && filterCount > 0) {
+    return i18n.translate(
+      'xpack.ml.importExport.exportFlyout.exportJobDependenciesWarningCallout.filterOnlyTitle',
+      {
+        defaultMessage:
+          '{jobCount, plural, one {# selected job has} other {# selected jobs have}} dependencies on {filterCount, plural, one {a filter} other {filters}}',
+        values: { jobCount: jobs.length, filterCount },
+      }
+    );
+  }
+
+  return i18n.translate(
+    'xpack.ml.importExport.exportFlyout.exportJobDependenciesWarningCallout.filterAndCalendarTitle',
+    {
+      defaultMessage:
+        '{jobCount, plural, one {# selected job has} other {# selected jobs have}} dependencies on filters and calendars',
```

Comment:
As above, maybe best keep with the term `filter lists` here.

Reply:
```suggestion
        '{jobCount, plural, one {# job uses} other {# jobs use}} calendars and filter lists',
```

Reply:
Updated in https://github.com/elastic/kibana/pull/107416/commits/e64c591042b919577dc0e9c81c1ed2e92b043579